velox/docs/functions/spark/string.rst

```diff
@@ -131,14 +131,24 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
 .. spark:function:: split(string, delimiter) -> array(string)
 
-    Splits ``string`` on ``delimiter`` and returns an array. ::
+    Splits ``string`` on ``delimiter`` and returns an array. 
+    Arguments:
+        ``string`` - a string expression to split, constant
+        ``delimiter`` - a string representing a regular expression, supported by re2.
+    Notice:
+        There are some usage not supported by re2 regex.  If used, there will be 
```

Comment:
I think we can remove this clarification. RE2 indeed has a few incompatible patterns with other libs. It may be better to just clarify them at some common place, instead of for only a certain regex function.